week1/homework/src/server.js

```diff
@@ -2,14 +2,35 @@
 
 const http = require('http');
 
-/* `createServer` MUST return an instance of `http.Server` otherwise the tests
- * will fail.
- */
 function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    let serverResponseObject = {};
+    response.setHeader('Content-Type', 'application/json');
+
+    if (request.url === '/state') {
+      serverResponseObject.state = state;
+    }
+ else if (request.url === '/add') {
```

Comment:
The indentation of the `else if` clauses are wrong. They should be aligned with the first `if`

Reply:
Thanks, I fixed it by ignoring prettier.